documentation/docs/02_account_manager.md

```diff
@@ -33,6 +33,34 @@ The wallet needs to be initialised (`init` command).
 $ ./wallet main
 ```
 
+### `./wallet backup`
+
+Creates a stronghold backup file.
+
+#### Parameters
+
+| Name    | Optional  | Example              |
+| ------- | --------- | -------------------- |
+| `path`  | ✘         | "backup.stronghold"  |
+
+#### Example
+
+Create a stronghold backup file.
+```sh
+$ ./wallet backup backup.stronghold
+```
+
+### `./wallet change-password`
+
+Changes the stronghold password.
+
+#### Example
+
+Change the stronghold password.
+```sh
+$ ./wallet change-password
```

Comment:
Isn't the current password missing here?

Reply:
No, it's being asked safely (hidden) at start